src/app/products/loan-products/loan-product-stepper/loan-product-settings-step/loan-product-settings-step.component.ts

```
@@ -282,435 +291,468 @@ export class LoanProductSettingsStepComponent implements OnInit {
-        }
-      });
-
-    this.loanProductSettingsForm
-      .get('useDueForRepaymentsConfigurations')
-      .valueChanges.subscribe((useDueForRepaymentsConfigurations: boolean) => {
-        if (useDueForRepaymentsConfigurations) {
-          this.loanProductSettingsForm.patchValue({
-            dueDaysForRepaymentEvent: null,
-            overDueDaysForRepaymentEvent: null
+          // Only Advanced Payment Allocation Strategy
+          this.transactionProcessingStrategyDataBase.some((cn: CodeName) => {
```

Biome (2.4.4)

[error] This callback passed to some() iterable method should always return a value.

(lint/suspicious/useIterableCallbackReturn)